Chapter4/42MinimalTree.py

```diff
@@ -16,7 +16,7 @@ def initiateArrayToBinary(array):
 def arrayToBinary(array, start, end):
     if start > end:
         return ''
-    mid = (start + end) / 2
+    mid = int(len(array) / 2)
```

Comment:
I think this will break the function since it needs to account for start and end?